actions/notification_actions.jsx

```diff
@@ -141,19 +141,20 @@ export function sendDesktopNotification(post, msgProps) {
         const activeChannel = getCurrentChannel(state);
         const channelId = channel ? channel.id : null;
         const notify = (activeChannel && activeChannel.id !== channelId) || !state.views.browser.focused;
+        const soundName = user.notify_props !== undefined && user.notify_props.desktop_notification_sound !== undefined ? user.notify_props.desktop_notification_sound : 'Bing';
 
         if (notify) {
-            dispatch(notifyMe(title, body, channel, teamId, !sound));
+            dispatch(notifyMe(title, body, channel, teamId, !sound, soundName));
```

Comment:
see my comment in [the other PR](https://github.com/mattermost/desktop/pull/1351#discussion_r458642132) that we might prefer to pass `data` around.

Reply:
Ok done ;)